.travis.yml

```diff
@@ -1,4 +1,5 @@
 sudo: true
+dist: trusty # needed for php 5.5 and smaller
```

Comment:
this seems to fix 5.4 and 5.5 but 5.3 still fails @mlocati any idea?

Reply:
I found the reason - fixed by https://github.com/avalanche123/Imagine/pull/744

Reply:
@mlocati Did rebase the PR